content/en/logs/guide/reduce_data_transfer_fees.md

```diff
@@ -0,0 +1,40 @@
+---
+title: How to send logs to Datadog while reducing data transfer fees
+further_reading:
+- link: "https://aws.amazon.com/solutions/case-studies/textnow-privatelink-case-study/"
+  tag: "AWS Case Study"
+  text: "TextNow Saves 93% on Data Transfer Fees Using AWS PrivateLink"
+- link: "/logs/log_configuration/flex_logs/#potential-sources-for-sending-directly-to-flex-logs"
+  tag: "Documentation"
+  text: "Potential Sources for Sending Directly to Flex Logs"
+algolia:
+  tags: ["data transfer", "data egress", "private link", "PrivateLink", "Private Service Connect"]
+---
+
+## Overview
+
+As your organization grows, the amount of data you transfer between cloud providers to Datadog can also increase.  Cloud providers charge a *data transfer* fee or *data egress* fee to move that data from cloud storage over public IP addresses. This can easily become one of the largest line items in your organization’s cloud cost bill. 
```

Comment:
```suggestion
As your organization grows, the amount of data you transfer between cloud providers to Datadog can also increase. Cloud providers charge a *data transfer* fee or *data egress* fee to move that data from cloud storage over public IP addresses. This can easily become one of the largest line items in your organization’s cloud cost bill. 
```